### PR TITLE
Update views 2

### PR DESCRIPTION
**Pull request by ritamma**
Updates:
- add calendar for services views with current user bookings
- update style for flash messages with Stimulus

### CODE REVIEW

**Review by evrusha — APPROVED** [review]

File: app/views/shared/_flash.html.erb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<% flash.each do |flash_type, message| %>
+  <div
+    class="flash__message"
+    data-controller="removals"
+    data-action="animationend->removals#remove"
```
Comment (evrusha) [inline app/views/shared/_flash.html.erb:5]: Where is removals_controller.js?

Comment (ritamma) [reply app/views/shared/_flash.html.erb:5]: Thanks! I forgot to add this file to the commit

File: app/controllers/services_controller.rb [hunk 2]
```diff
@@ -5,7 +5,7 @@ class ServicesController < ApplicationController
   before_action :authorize_service
 
   def index
-    @pagy, @services = pagy((@company.services.all), items: 1)
+    @pagy, @services = pagy((@company.services.where(status: 'active').all), items: 1)
```
Comment (fooooxmr) [inline app/controllers/services_controller.rb]: add default scope or services.active

Comment (ritamma) [reply app/controllers/services_controller.rb]: Done

**Pull request merged**